packages/core/src/lib/modal.svelte

```diff
@@ -26,81 +26,84 @@ Creates a modal overlay.
 
 <script lang="ts">
 import cx from 'classnames';
-import { createEventDispatcher, onMount } from 'svelte';
+import { createEventDispatcher } from 'svelte';
 import IconButton from './button/icon-button.svelte';
+import { clickOutside } from '$lib';
 
 /** Whether the modal is open. */
 export let open = false;
+
+$: if (typeof document !== 'undefined') {
+  document.body.classList.toggle('overflow-hidden', open);
+}
+
 /** The variant of the modal. */
 export let variant: 'small' | '' = '';
 
 const dispatch = createEventDispatcher<{
   /** Fires when the modal is requested to close. */
-  close: boolean;
+  close: void;
```

Comment:
`void` is only a valid type annotation for function returns. I think this should be `undefined` or `never`. If TS complains about either of those, then I think `close: true` and not worrying about it any more is also fine

Reply:
i was able to do `null` here, not sure if that is any better though